myVK/myVK/ViewController/Controller/Collection/FriendPhotoCollectionViewController.swift

```diff
@@ -5,25 +5,37 @@ import UIKit
 
 /// экран с фотографиями друга
```

Comment:
```suggestion
/// Экран с фотографиями друга
```
(Зачем тебе фотографии твоего друго?:-))